pkg/cmd/operator/webhooks.go

```diff
@@ -105,6 +133,10 @@ func (o *WebhookOptions) Validate() error {
 		return errors.New("port can't be zero")
 	}
 
+	if len(o.Validators) == 0 {
```

Comment:
Validate is meant to validate the options coming from the flags. This seems like it should fail in `NewWebhookCmd` where it is supplied.

Reply:
Added a default instead of an error in this case.

Reply:
I think this should fail. If you accidentally give it an empty array and it will silently use a global variable instead - that feels wrong IMO.

Reply:
Erroring would result in carrying it up to a main. None of our command initialization returns error so far. 
We could panic, as this is clearly a developer fault, but I think default is reasonable.

Reply:
I can live with a panic but it shouldn't silently use something else.

Reply:
changed to panic